test/testdata/parser/whitequark/assert_parses_args_10.rb

```diff
@@ -1,4 +0,0 @@
-# typed: true
-
-def f foo:
```

Comment:
ditto

Reply:
this guy moved to `kwarg_no_paren`. `whitequark/parser` has a few utility methods for testing and `assert_parses_args` is one of them. The name of the real test is `test_kwarg_no_paren`